extensions/interactions/SetInput/SetInput.py

```diff
@@ -38,7 +38,14 @@ class SetInput(base.BaseInteraction):
 
     # NB: There used to be a UnicodeString-typed parameter here called
     # 'element_type'. This has since been removed.
-    _customization_arg_specs = []
+    _customization_arg_specs = [{
+        'name': 'buttonText',
+        'description': 'Customize the "Add" button text:',
```

Comment:
This description should be descriptive of what the field represents, not an instruction to the creator. See other interactions for examples.

Reply:
Done!